### PR TITLE
github description added

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: 01/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+¿Qué es GitHub?
+Es un repositorio online gratuito que permite gestionar proyectos y controlar versiones de código. Es muy utilizado por desarrolladores para almacenar sus trabajos dando así la oportunidad a millones de personas de todo el mundo a cooperar en ellos.
```
Comment (eamzea) [inline 01/index.html:10]: Online repository?

File: 01/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Permite alojar proyectos utilizando el sistema de control de versiones Git. 
+¿Para qué sirve?
+Se utiliza principalmente para la creación de código fuente de programas de ordenador.
+¿Qué es un PR?
+Una Pull request funciona para solicitar que otro desarrollador (por ejemplo, el mantenedor del proyecto) incorpore (o haga un pull) una rama de tu repositorio al suyo.
```
Comment (eamzea) [inline 01/index.html:15]: And in your words?
Able to make a pull from other repot to other one?

This questions were created to allow you to try to explain what you understood.

Read it and try to understand what you wrote. You can do as exercise trying to explain it to a 5 years old child

File: 01/index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+¿Qué es un merge?
+Permite tomar las líneas independientes de desarrollo creadas por git branch e integrarlas en una sola rama.
```
Comment (eamzea) [inline 01/index.html:17]: Again, try using your own words. We don't want a dictionary definition. We want to know if you understood the main idea.

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — APPROVED** [review]
LGTM!

Just needs to be updated

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**